ios/Helium/Info.plist

```diff
@@ -37,22 +37,18 @@
 			</dict>
 		</dict>
 	</dict>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Allow Helium to use your location to add your Helium Hotspots to the network</string>
-	<key>NSFaceIDUsageDescription</key>
-	<string>Allow Helium to authenticate with Face ID</string>
 	<key>NSBluetoothAlwaysUsageDescription</key>
 	<string>Allow Helium to connect to and manage your Hotspots</string>
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Allow Helium to connect to and manage your Hotspots</string>
-	<key>UIBackgroundModes</key>
```

Comment:
Did we want to remove this? I think it was for push notifications?

Reply:
No, whoops, I don't know why these things got removed. I just intended to remove the landscape modes.